cmd/cli/client/pdp/proofset/list.go

```diff
@@ -0,0 +1,46 @@
+package proofset
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/storacha/piri/pkg/config"
+	"github.com/storacha/piri/pkg/pdp/httpapi/client"
+)
+
+var (
+	ListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "List all proof sets",
+		Args:  cobra.NoArgs,
+		RunE:  doList,
+	}
+)
+
+func doList(cmd *cobra.Command, _ []string) error {
+	ctx := cmd.Context()
+
+	cfg, err := config.Load[config.PDPClient]()
+	if err != nil {
+		return fmt.Errorf("loading config: %w", err)
+	}
+
+	api, err := client.NewFromConfig(cfg)
+	if err != nil {
+		return fmt.Errorf("creating client: %w", err)
+	}
+
+	proofSet, err := api.ListProofSet(ctx)
+	if err != nil {
+		return fmt.Errorf("getting proof set status: %w", err)
+	}
+	jsonProofSet, err := json.MarshalIndent(proofSet, "", "  ")
```

Comment:
Is there a more friendly format that can be output by default? I feel like you should request JSON output if that's what you want.

Reply:
Yeah this can be improved to format the output as a table, for example. Treating that as out of scope for now.